regex-assembly/include/sql-injection-function-names.ra

```diff
@@ -1,9 +1,6 @@
 ##! Please refer to the documentation at
 ##! https://coreruleset.org/docs/development/regex_assembly/.
 
-##!^ \b
```

Comment:
Was it really necessary to move the prefix and suffix out of the file?

Reply:
Yes, it was necessery. The generated regex, when the suffix and prefix were in the file, was different and the tests failed. Therefore I moved them out of the file. Does that make sense?

Reply:
Regex when generated when prefix and suffix IN the file:
https://regex101.com/r/UnEfRc/1

Regex when generated when prefix outside of the file:
https://regex101.com/r/LlgOkp/1

I didn't investigate further but solved it with moving the prefix and suffix out of the file.

Reply:
It does make sense, I didn't implement that for simplicity. Would have come in handy now though...